commands/kubernetes.go

```diff
@@ -1557,6 +1611,15 @@ func buildNodePoolCreateRequestFromArgs(c *CmdConfig, r *godo.KubernetesNodePool
 	return nil
 }
 
+func buildClusterRegistryRequestFromArgs(c *CmdConfig, r *godo.KubernetesClusterRegistryRequest) error {
+	uuids, err := c.Doit.GetStringSlice(c.NS, doctl.ArgClusterUUID)
```

Comment:
We should either make the UUID flag required or (if we use an arg like suggested above) check the user input here and return an error if no UUID has been passed.

Reply:
@andrewsomething - decided to go with args.